src/components/HomePage/SearchArea/ShowAdvancedOptionsButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Fab from '@material-ui/core/Fab';
+import Close from '@material-ui/icons/CloseRounded';
+import MoreHoriz from '@material-ui/icons/MoreHoriz';
```

Comment:
Use tree-shaking for imports (I believe our configs allow this) - it's suggested in the MUI docs.

Reply:
This but everywhere, just noticed it

Reply:
Tree shaking must be configured at webpack level, since we created the project with create-react-app, I am not sure if it is possible to change now

Reply:
And isn't it enabled already? Can you test including something in the "regular" way? I have as reference a gatsby project with which it worked straight away - not much but something

Also, having our own webpack configs also looks like something desireable, as I believe that some useful configs are facilitated when using it directly, i.e after ejecting from `create-react-app` (such as custom stuff for custom file extensions, might be useful for translations when that happens in the far, far, far future).

Reply:
After doing some quick research, I noticed create-react-app already supports tree-shaking without having to eject and change webpack configs. I will look into how to refactor the code to use this feature

Reply:
I can help by pointing out stuff. But generally, it will be something like changing

```js
import Close from '@material-ui/icons/CloseRounded';
import MoreHoriz from '@material-ui/icons/MoreHoriz';
```

to

```js
import { Close, MoreHoriz } from '@material-ui/icons';
```

You can probably use `grep` or `vscode` to search in the whole project for this - possible candidates are `@material-ui/core` and `@material-ui/icons`, also maybe `@material-ui/styles` if it's being used somewhere.
I'm thinking some of the `eslint` rules added in #22 might help with this, namely ones that add checks for duplicated imports and such

Reply:
It is already done, Regex is <3. I am assuming all tree-shaking configuration is already provided by create-react-app boilerplate, as I've read on the internet, and if it on the internet, it must be true, so we are good.

Reply:
If it still works after the code change on your machine (and Netlify's PR preview) it's probably fine

:shipit: